sa/storage-authority.go

```diff
@@ -673,6 +695,45 @@ func (ssa *SQLStorageAuthority) FinalizeAuthorization(authz core.Authorization)
 	return
 }
 
+// RevokeAuthorization invalidates a pending or finalized authorization
+func (ssa *SQLStorageAuthority) RevokeAuthorization(authz core.Authorization) (err error) {
```

Comment:
I think authorization objects are big enough we should probably take them as pointers as a matter of habit. But this is rarely called so I'm willing to be convinced it's not necessary here!